src/is-promise.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const isPromise = <T>(obj: any): obj is Promise<T> => {
```

Comment:
For the sake of consistency with the rest of the package, please`export` here instead of exporting an object at the end of the file.